### DISCUSSION TITLE
SDR++ Doesn't see my RSP1

### DISCUSSION THREAD

**Discussion by ND1L** [AlexandreRouma/SDRPlusPlus#696] [Q&A]
I downloaded SDR++ this evening to a new laptop. I had never run any other SDR client on this computer, and when I started SDR++ it didn't see the RSP1.  This is a Win10 laptop with abundant RAM & disk space.

Help!!

Jess ND1L

**Comment by AlexandreRouma** [comment]
You have to install the SDRplay API and make sure the API service is running.
Follow the instructions on their website

**Comment by ND1L** [comment]
Got it, thanks!  As it happens, I installed SDR UNO which is recommended for my RSP1. That installed the SDRplay API as part of the installation process, so it is now working correctly. It’s been so long since I last installed anything for the SDRplay I’ve forgotten all the ins and outs of setting it up. I’m switching to a new laptop so I’m running into similar issues with lots of app’s.

 

Thanks again!
